### PR TITLE
fix compiler warnings seen on Travis CI, for the Linux build, and in MinGW

### PR DESCRIPTION
**Pull request by Jojo-Schmitz**

### CODE REVIEW

File: libmscore/layout.cpp [hunk 1]
```diff
@@ -2553,8 +2553,10 @@ void layoutDrumsetChord(Chord* c, const Drumset* drumset, const StaffType* st, q
 std::pair<qreal, qreal> extendedStemLenWithTwoNoteTremolo(Tremolo* tremolo, qreal stemLen1, qreal stemLen2)
       {
       const qreal spatium = tremolo->score()->spatium();
+#if 0
       const qreal sw = tremolo->score()->styleS(Sid::tremoloStrokeWidth).val();
       const qreal td = tremolo->score()->styleS(Sid::tremoloDistance).val();
+#endif
```
Comment (Howard-C) [inline libmscore/layout.cpp:2559]: How about moving these lines down to join the other disabled block of code?

Comment (Jojo-Schmitz) [reply libmscore/layout.cpp:2559]: Good call. Will do.

Comment (Jojo-Schmitz) [reply libmscore/layout.cpp]: MinGW complains about those too BTW

File: libmscore/layout.cpp [hunk 2]
```diff
@@ -2579,6 +2577,9 @@ std::pair<qreal, qreal> extendedStemLenWithTwoNoteTremolo(Tremolo* tremolo, qrea
 #if 0
       // cross-staff & beam between staves: extend both stems by the same length
       else if (tremolo->crossStaffBeamBetween()) {
+            const qreal sw = tremolo->score()->styleS(Sid::tremoloStrokeWidth).val();
+            const qreal td = tremolo->score()->styleS(Sid::tremoloDistance).val();
+
```
Comment (Spire42) [inline libmscore/layout.cpp:2582]: This newline seems like it shouldn't be here.

Comment (Jojo-Schmitz) [reply libmscore/layout.cpp:2582]: separates variables from the code using them. Is done that way in about a million other places

Comment (Jojo-Schmitz) [reply libmscore/layout.cpp:2582]: Hhmmm. OK not here though, I see now

Comment (Jojo-Schmitz) [reply libmscore/layout.cpp]: fixed

**Pull request merged**